README.md

```diff
@@ -133,7 +135,8 @@ Additionally, `cowboy_swagger` can be configured/customized from a `*.config` fi
    {global_spec,
     #{swagger => "2.0",
       info => #{title => "Example API"},
-      basePath => "/api-docs"
+      %% See "path" config section
+      basePath => "/api-docs-swagger"
```

Comment:
Same here, I don't see why the base path (which is `api-docs` for all the projects I use this with) would be different from what it was. Can we restore it?